WalletWasabi.Documentation/Guides/DeterministicBuildGuide.md

```diff
@@ -1,68 +1,90 @@
-> Reproducible [or deterministic] builds are a set of software development practices that create an independently-verifiable path from source to binary code.- https://reproducible-builds.org/
+# Guide for deterministic builds
 
-This guide describes how to reproduce Wasabi's builds. If you got stuck with these instructions, take a look at how to build Wasabi from source code: https://github.com/zkSNACKs/WalletWasabi#build-from-source-code
+The term *deterministic builds* is [defined](https://reproducible-builds.org/) as follows:
 
-# 1. Assert Correct Environment
+> Reproducible [or deterministic] builds are a set of software development practices that create an independently-verifiable path from source to binary code.
 
-In order to reproduce Wasabi's builds you need Git, Windows 10 and the version of .NET Core SDK that was the most recent in the time of building the release.
+This guide describes how to reproduce Wasabi's builds. If you get stuck with these instructions, take a look at [how to build Wasabi from source code](https://docs.wasabiwallet.io/using-wasabi/BuildSource.html).
 
-# 2. Reproduce Builds
+**Warning:** Reproducible builds were introduced in [1.1.3 release](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v1.1.3), you cannot use these instructions for older versions!
+
+## 1. Assert correct environment
+
+In order to reproduce Wasabi's builds, you need [git](https://git-scm.com/) package, Windows 10, and the version of .NET Core SDK that was used by the Wasabi team to produce given Wasabi Wallet release. The latest version of .NET Core SDK is always used, unless specified otherwise in the release notes of Wasabi Wallet. You can get it here https://dotnet.microsoft.com/download/dotnet-core.
```

Comment:
```suggestion
In order to reproduce Wasabi's builds, you need [git](https://git-scm.com/) package, Windows 10, and the version of [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core) that was used by the Wasabi team to produce the release. The latest version of .NET Core SDK is always used, unless specified otherwise in the release notes of Wasabi Wallet.
```